demo/pkg/injector/ws.go

```diff
@@ -0,0 +1,14 @@
+package injector
+
+import (
+	"context"
+
+	"github.com/99designs/gqlgen/graphql/handler/transport"
+)
+
+func InitPayloadFunc(ctx context.Context, initPayload transport.InitPayload) (context.Context, *transport.InitPayload, error) {
```

Comment:
Is this only used for testing? If so, please put it in a test package.

Reply:
No, it needs to be able to run in the demo subgraphs as a normal program, otherwise you can't run them standalone.